<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BtreeWindow</class>
 <widget class="QWidget" name="BtreeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="testBrowser">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>256</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>70</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PreOrdTra">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>先序遍历</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InOrdTra">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>中序遍历</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PostOrdTra">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>后序遍历</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CountLN">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>统计叶子结点数目</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InOrdThr">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>中序线索化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PreOrdThr">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>先序线索化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PostOrdThr">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>后序线索化</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
